redom/android/src/main/java/com/pushtorefresh/redom/android/view/EditTextImpl.kt

```diff
@@ -0,0 +1,43 @@
+package com.pushtorefresh.redom.android.view
+
+import androidx.appcompat.widget.AppCompatEditText
+import com.jakewharton.rxbinding2.view.RxView
+import com.jakewharton.rxbinding2.widget.RxTextView
+import com.pushtorefresh.redom.api.Component
+import com.pushtorefresh.redom.api.EditText
+import com.pushtorefresh.redom.api.View
+import com.pushtorefresh.redom.api.toViewStructure
+import io.reactivex.Observable
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.disposables.Disposable
+
+class EditTextImpl<O : Any> : EditText<O>, TextViewImpl<O>() {
+
+    override fun build(): Component<O, *> {
+        return EditTextComponent(
+                this,
+                EditText::class.java,
+                Observable.merge(outputObservables)
+        )
+    }
+
+    private class EditTextComponent<O : Any>(
+            private val editTextImpl: EditTextImpl<O>,
```

Comment:
editText is mutable, it means someone can mutate output or changeText for example, do we need that behavior?

Reply:
This raises a good question, I'll inline fields for now but that's a good point